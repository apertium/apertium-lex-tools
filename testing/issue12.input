^c<a>/c<a>$ ^w<n>/want<n>/got<n>$
