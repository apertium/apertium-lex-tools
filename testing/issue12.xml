<rules>
  <rule weight="2.0">                    <!-- rule1 -->
    <or>
      <match lemma="c"/>                 <!-- A -->
      <match tags="unseen"/>             <!-- B -->
    </or>
    <match lemma="salkdfj" tags="adv.*"> <!-- C -->
      <select lemma="lksajfdlk"/>
    </match>
  </rule>

  <rule><match lemma="w" tags="n"><select lemma="want"/></match></rule>
  <rule weight="2.0">                    <!-- rule2 -->
    <match tags="unseen"/>               <!-- B -->
    <match lemma="w" tags="n">
      <select lemma="got"/>
    </match>
  </rule>
</rules>
