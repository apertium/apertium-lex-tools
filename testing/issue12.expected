^c<a>/c<a>$ ^w<n>/want<n>$
